packages/libs/react-ui/src/styles/vars.css.ts

```diff
@@ -53,7 +53,8 @@ export const vars = createGlobalTheme(':root', {
   },
   shadows: {
     // TODO: Update to match design system
-    $1: `0px 1px 2px 0 $colors$gray40`,
+    $1: `0px 1px 0 0 ${colorPalette.$gray30}`,
+    $2: `0px 1px 0 0 ${colorPalette.$gray60}`,
```

Comment:
What is the reason for adding these to the vars.css file?

Reply:
the reason is that we probably sould predefine our shadows here and reuse them, so i added one for each theme, i can also not do it and specify it manually in the Form css

Reply:
If it's used somewhere else as well, I think it make sense to add it here, but then it would probably have a more descriptive token name. In this case I think it is specific to the input components so i would define it there

Reply:
yeah i was thinking on giving it shadow input related name

Reply:
Shall remove these variables completely since they aren't in the sprinkles file anymore?